### PR TITLE
add SUPPORT file for GitHub

### LINKED ISSUE #6323: add SUPPORT file
It should at least help us point to https://talk.jekyllrb.com for usage support.
Also mention https://jekyllrb.com/help/ for getting help.

Anything else we should mention in this file @jekyll/affinity-team-captains ?

see: https://help.github.com/articles/adding-support-resources-to-your-project/

👋  @benbalter Could you point to an existing good SUPPORT file example for inspiration?

### PR DESCRIPTION
**Pull request by DirtyF**
fix #6323

### CODE REVIEW

**Comment by ashmaroli** [comment]
stackoverflow..? or is that a given?

File: .github/SUPPORT.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Jekyll Support
+
+## Getting Help
+
+If you're looking for support for Jekyll there are a lot of options, please check out:
```
Comment (benbalter) [inline .github/SUPPORT.md]: Would it be worth being specific here, e.g., "Jekyll's issue tracker is not a support forum"?

Comment (DirtyF) [inline .github/SUPPORT.md]: Fixed. How could I forget about that?  🐵

File: .github/SUPPORT.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+* Read [Jekyll Documentation](https://jekyllrb.com/docs/home/)
+* If you have a question about using Jekyll, start a discussion on [Jekyll Forum](https://talk.jekyllrb.com/) or [StackOverflow](https://stackoverflow.com/questions/tagged/jekyll)
+* Chat with Jekyllers &mdash; Join [our Gitter channel](https://gitter.im/jekyll/jekyll) or [our IRC channel on Freenode](irc:irc.freenode.net/jekyll)
+
+There are a bunch of helpful community members on these services that should be willing to point you in the right direction.
+
+## Report a bug
+
+* If you think you've found a bug within a Jekyll plugin, open an issue in that plugin's repository.
```
Comment (parkr) [inline .github/SUPPORT.md]: How do they know where to find the repository? Generally https://rubygems.org/gem/plugin-name then click `Homepage` is what I do personally. We should help the user a little here.

**Comment by DirtyF** [comment]
@jekyllbot: merge+docs

**Pull request merged**